utils/audio.js

```diff
@@ -1,18 +1,101 @@
 'use strict';
 
-module.exports = {
-  buildSfxCollection(game, componentName, clipCount) {
-    // FIXME: Make this a class.
+class Audio {
```

Comment:
Not sure if you made this a class in response to the FIXME just below, but that was actually referring to the `sounds` array. Sort of doesn't make sense for this util to be a class to me, since it's all static methods.